react/the_react_ecosystem/project_shopping_cart.md

```diff
@@ -0,0 +1,41 @@
+### Introduction
+
+By now you've come far from your React-baby days, with tools like routers and testing frameworks under your belt. You have an even longerou still have a longer way to go. It's a great time to put these concepts to use with a classic project - a mock shopping cart.
+
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+1.  Create a new React Project.
+2.  Think about the component and the folder structure. How could you set up your application? Which components or functionalities do you need? It's a good idea to note this down somewhere you can easily get to and refer back and add to it to keep track.
+3.  You should have at least two pages (a homepage and a shop page, which includes your shopping cart). Let a user navigate between the pages with a navigation bar, which will be shown on both routes.
+4.  To your homepage, you can add whatever you'd like! A few images or information will be totally fine; it doesn't have to be something fancy - it's to test the concepts taught thus far.
+5.  The same navigation bar should show up on the shopping cart route, which displays the number of items currently in the cart. You should also have a button next to it where you can go to the cart to checkout and pay (however we are not going to implement this logic here).
```

Comment:
```suggestion
5.  On the shopping cart page, you should have the same navigation bar that displays the number of items currently in the cart. You should also have a button next to it where you can go to the cart to checkout and pay (however we are not going to implement this logic here).
```

Also a small nitpick, I think it's better to keep using `page(s)` instead `route(s)`, what do you think?